WooCommerce/Classes/ViewModels/Refunds/RefundedProductsDataSource.swift

```diff
@@ -146,6 +146,15 @@ extension RefundedProductsDataSource {
 
         sections = [refundedProducts].compactMap { $0 }
     }
+
+    /// Handle taps on cells
+    ///
+    func tableView(_ tableView: UITableView,
+                   in viewController: UIViewController,
+                   didSelectRowAt indexPath: IndexPath) {
```

Comment:
are the diffs here trying to disable table view cell tap actions? if so, the cells are still tappable from testing that navigate to a Product modal. It looks like `RefundedProductsViewController` is the class that implements `UITableViewDelegate` that handles the tap action: https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/ViewRelated/Orders/Order%20Details/Payment%20Section/Refunds/RefundedProductsViewController.swift#L152-L156

alternatively, you can set [`UITableView`'s `allowsSelection` property](https://developer.apple.com/documentation/uikit/uitableview/1614911-allowsselection) to false to disable cell tap action for all the rows